lib/challenge.js

```diff
@@ -20,6 +20,18 @@
 
 // Answer Starts Here
 
+'Calendar'
+Description: Our calendar program tracks activities, events and to-dos
+every calendar has
+  - hours
+  - days
+  - months
+  - years
+every calendar can
+  - schedule appointments/events
+  - list to-do activities
+  - set an alert to notify you of an upcoming event
+
```

Comment:
Well done, this is what was asked for. The question asked for 3 of these, though, so I can only give you half credit.